install/configs/karaf/islandora_collection.script

```diff
@@ -1,2 +1,2 @@
-repo-add file:/home/vagrant/islandora/camel/karaf/target/classes/features.xml
-feature:install islandora-collection-service
\ No newline at end of file
+repo-add file:/home/vagrant/Alpaca/karaf/target/classes/features.xml
+feature:install islandora-collection-service
```

Comment:
Just a thought/Question, but until we have micro service (including drupal connection to it) working that can replace this, we should keep the old functionality to not break the currently-semi-working behaviour we have at future.islandora.ca?